foodnet/membership/tests.py

```diff
@@ -1,23 +1,79 @@
 # -*- coding: utf-8 -*-
 from django.test import TestCase
 from django.core.urlresolvers import reverse
-from django.contrib.auth.models import User
+from django.contrib.auth import get_user_model
 
 
 class TestMembership(TestCase):
     def setUp(self):
-        self.user = User.objects.create(
-            username='testuser',
-            email='test@food.net'
+        self.user = get_user_model().objects.create_user(
+            'test@food.net', is_active=True
         )
+        self.user.set_password('pass')
         self.user.save()
+        
+        self.assertEqual(1, get_user_model().objects.all().count())
```

Comment:
Is it good practice to make assertions in `setUp()`? Isn't it just for, yes, settings things up before doing the actual testing? I'd probably have this inside a `test_*` method.